expression/util.go

```diff
@@ -449,3 +450,19 @@ func (s *exprStack) push(expr Expression) {
 func (s *exprStack) len() int {
 	return len(s.stack)
 }
+
+// ColumnSliceIntersect intersects two column slice.
+// You need to make sure that at least each element in s2 is unique.
+func ColumnSliceIntersect(s1, s2 []*Column) []*Column {
```

Comment:
The caller only wants to know if there is any intersect, so just return `bool` is enough.

Reply:
Agree, changed.